package.json

```diff
@@ -4,8 +4,8 @@
   "description": "IOTA Flash Payment Channels",
   "main": "./lib/flash.js",
   "scripts": {
-    "test": "mocha",
-    "build": "gulp"
+    "test": "node_modules/mocha/bin/mocha",
+    "build": "node_modules/gulp/bin/gulp.js"
```

Comment:
I don't see a reason to specify the full path here. It does work with the normal process.